examples/deployment/kubernetes/deploy.sh

```diff
@@ -45,22 +42,23 @@ gcloud auth configure-docker
 go get github.com/google/trillian/...
 cd $GOPATH/src/github.com/google/trillian
 
```

Comment:
Could have a list of targets (`log_server`, `log_signer`, `map_server`) that each of the stages iterates over.
(Can defer to a future PR)